src/Session.ts

```diff
@@ -78,15 +112,29 @@ export class Session extends EventEmitter {
     this.emit("login");
   };
 
+  /**
+   * Fetches data using available login information. If the user is not logged in, this will behave as a regular `fetch`. The signature of this method is identical to the [canonical `fetch`](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API).
+   *
+   * @param url The URL from which data should be fetched.
+   * @param init Optional parameters customizing the request, by specifying an HTTP method, headers, a body, etc. Follows the [WHATWG Fetch Standard](https://github.github.io/fetch/).
```

Comment:
The other link is to the polyfill :)

```suggestion
   * @param init Optional parameters customizing the request, by specifying an HTTP method, headers, a body, etc. Follows the [WHATWG Fetch Standard](https://fetch.spec.whatwg.org/).
```